Cannot find module '@nx/react/tailwind'

when run
`cd apps/renderer && bun run dev`

error：
```
$ vite

  VITE v6.2.3  ready in 214 ms

  ➜  Local:   http://localhost:4200/
  ➜  press h + enter to show help
5:45:59 pm [vite] Internal server error: [postcss] Cannot find module '@nx/react/tailwind'
```

--- a/package.json
+++ b/package.json
@@ -15,6 +15,7 @@
     "@biomejs/biome": "1.9.4",
     "@commitlint/cli": "^19.0.0",
     "@commitlint/config-conventional": "^19.0.0",
+    "@nx/react": "^20.8.0",
     "@swc-node/register": "~1.9.1",
     "@swc/core": "~1.5.7",
     "@swc/helpers": "~0.5.11",

